.changeset/sour-spiders-reply.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kadena/react-ui": patch
+---
+
+Feat/explicit imports
```

Comment:
When adding changesets I think we should be more descriptive of the change, otherwise it's better to add empty changesets instead. Since this doesn't actually change anything for consumers I would actually add an empty changset instead.

I think of these descriptions as what you would see on npm for the version update

Reply:
I admit I've just been using the bot's suggested changesets! I'll have a look :D